### DISCUSSION TITLE
Release 2.3.4?

### DISCUSSION THREAD

**Discussion by the-spyke** [debauchee/barrier#1300] [Q&A] [answered]
It has been more that a year since the last 2.3.3 release. May we have a new one?

It would be great to just install from a `.dmg` on macOS instead of building from source.

**Comment by TFWol** [comment]
A dialog was started in #1271

**Comment by shymega** [comment, marked as answer]
v2.3.4 has been released, which has *critical* security fixes. The project's recommendation is to upgrade to v2.3.4 at the very least, or alternatively, v2.4.0 has also been released, which has the latest commits at the time of release. I'm marking this response as the answer.
